lib/parse.js

```diff
@@ -87,6 +87,10 @@ var parseValues = function parseQueryStringValues(str, options) {
             val = val.split(',');
         }
 
+        if(part.indexOf('[]=') > -1){
+          val = Array.isArray(val) ? [val] : val
```

Comment:
qs supports ES3, so we can't rely on `Array.isArray` being there

Reply:
@ljharb 
replaced in utils
`var isArray = Array.isArray;` this is being used in the utils file
with
`var isArray = Object.prototype.toString.call(obj) === '[object Array]';`